### PR TITLE
Building cart component

### PR DESCRIPTION
**Pull request by hassanidris**

### CODE REVIEW

**Comment by hassanidris** [comment]
@Quantinx If there are conflict I will resolve it just review it

File: client/redux/cartReducer.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    //   if (item) {
+    //     item.quantity += action.payload.quantity;
+    //   } else {
+    //     state.push(action.payload);
+    //   }
+    // },
+
+    addToCart: (state, action) => {
+      console.log(action);
```
Comment (Quantinx) [inline client/redux/cartReducer.js:22]: missed log

File: client/redux/cartReducer.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      const item = state.products.findIndex(
+        (item) => item.id === action.payload.id
+      );
+      console.log(item);
```
Comment (Quantinx) [inline client/redux/cartReducer.js:26]: log

**Review by Quantinx — APPROVED** [review]
Just need to clear up a few console logs in the next few merges , nothing major

**Comment by hassanidris** [comment]
I did it in this one and I will merge

**Pull request merged**